.circleci/config.yml

```diff
@@ -654,17 +100,8 @@ workflows:
   version: 2
   default-workflow:
     jobs:
-      - addons-versions-files-ratings
-      - amo-lib-locales-and-signing
-      - assets
-      - codestyle
-      - devhub
-      - main
       # Uncomment if you want to test the docker build
-      # - build-image
-      - reviewers-and-zadmin
-      - es-tests
-      - localization
+      - build-image
```

Comment:
Don't forget to re-comment this

Reply:
Cannot wait to remove that job so I don't have to keep enabling/disabling it.